packages/docusaurus-types/src/index.d.ts

```diff
@@ -419,6 +419,7 @@ interface HtmlTagObject {
   innerHTML?: string;
 }
 
+// TODO weird useless type, refactor
 export type ValidationResult<T> = T;
```

Comment:
I feel like this could be useful, e.g. In the future it could be `{normalizedObj: T, messages: string[]}`

Reply:
Agree but at the same time it would be a breaking change for our plugin ecosystem 😅